examples/sim.py

```diff
@@ -30,7 +30,8 @@ def future_loading(t, x=None):
     # simulate for 200 seconds
     print('\n\n------------------------------------------------')
     print('Simulating for 200 seconds\n\n')
-    (times, inputs, states, outputs, event_states) = batt.simulate_to(200, future_loading, print = True, progress = True)
+    simulated_results = batt.simulate_to(200, future_loading, print = True, progress = True)
+    simulated_monotonicity = simulated_results.states.monotonicity()
```

Comment:
As above, and with all future ones- can we change this to event_states. 

Also, can you print the results?